3.3/.sti/bin/assemble

```diff
@@ -1,23 +1,66 @@
 #!/bin/bash
 
-# For SCL enablement
-source .bashrc
+# This script uses these environment variables:
+#
+# DISABLE_COLLECTSTATIC: if not empty, inhibits execution of 'manage.py collectstatic'.
 
-set -e
+function assemble() {
+  # For SCL enablement
+  source .bashrc
```

Comment:
@mfojtik I'm saying it's not readable having an assemble function inside assemble script. I'm saying that having functions that group certain functionality is ok and the kind of "main" (which should not be a function) at the end which is showing nice flow of the script using those previously defined functions.